docs/api/inputs/chat.md

```diff
@@ -258,3 +258,83 @@ configuration with a dictionary conforming to the config.
 ```{eval-rst}
 .. autoclass:: marimo.ai.ChatAttachment
 ```
+
+## Supported Model Providers
+
+You can refer to the [LiteLLM page](https://litellm.vercel.app/docs/providers) for a list of OpenAI API supported providers. If you want any specific provider added explicitly (ones that don't abide by the standard OpenAI API format), you can file a [feature request](https://github.com/marimo-team/marimo/issues/new?template=feature_request.yaml).
+
+Normally, overriding the `base_url` parameter should work. Here are some examples:
+
+::::{tab-set}
+:::{tab-item} Cerebras
+
+```python
+chatbot = mo.ui.chat(
+   mo.ai.llm.openai(
+       model="llama3.1-8b",
+       api_key="csk-...", # insert your key here,
+       base_url="https://api.cerebras.ai/v1/",
+   ),
+)
+chatbot
+```
+
+:::
+:::{tab-item} Groq
+
+```python
+chatbot = mo.ui.chat(
+   mo.ai.llm.openai(
+       model="llama-3.1-70b-versatile",
+       api_key="gsk_...", # insert your key here,
+       base_url="https://api.groq.com/openai/v1/",
+   ),
+)
+chatbot
+```
+
+:::
+::::
```

Comment:
can you add an entry for xai? then we can close out that ticket #2773 
```
model="grok-beta",
base_url="https://api.x.ai/v1",
```

Reply:
i pulled this from their docs, i think its ok to not test and assume their docs are accurate in that case